packaging/rpm/cvmfs-universal.spec

```diff
@@ -83,6 +83,13 @@ BuildRequires: pkgconfig
 BuildRequires: python-devel
 BuildRequires: unzip
 
+%if 0%{?el4} || 0%{?el5} || 0%{?el6}
+BuildRequires: python2-devel
```

Comment:
Is `python2-devel` and `python3-devel` different from `python-devel` (required a few lines above)?